documentation/DATA_FORMAT.md

```diff
@@ -0,0 +1,374 @@
+        ...
+    ]
+}
+```
+
+<details>
+<summary><b>[Show me an example!]</b></summary>
+
+---
+
+For example, if a dataset had the following tracks and outliers, the file might look something like this.
+
+| Track # | Object IDs | Outliers |
+| ------- | ---------- | -------- |
+| 1       | 0, 1, 4    | 1        |
+| 2       | 2, 3, 5    | 2, 5     |
+
+```
+--outliers.json--
+{
+    "data": [
+        false, // 0
+        true,  // 1
+        true,  // 2
+        false, // 3
+        false, // 4
+        true   // 5
+    ]
+}
+```
+
+---
+
+</details>
+
+## Collections
+
+Collections are defined by an optional JSON file and group one or more datasets for easy access. Nucmorph-Colorizer can parse collection files and present its datasets for easier comparison and analysis from the UI.
+
+By default, collection files should be named `collection.json`.
+
+Collections are an array of JSON objects, each of which define the `name` (an **alias**) and the `path` of a dataset. This can either be a relative path from the location of the collection file, or a complete URL.
+
+If the path does not define a `.json` file specifically, Nucmorph-Colorizer will assume that the dataset's manifest is named `manifest.json` by default.
+
+```
+--collection.json--
+[
+    { "name": <some_name_1>, "path": <some_path_1>},
+    { "name": <some_name_2>, "path": <some_path_2>},
+    ...
+]
+```
+
+<details>
+<summary><b>[Show me an example!]</b></summary>
+
+---
+
+For example, let's say a collection is located at `http://example.com/data/collection.json`, and the `collection.json` contains this:
+
+```
+--collection.json--
+[
+  { "name": "Mama Bear", "path": "mama_bear" },
+  { "name": "Baby Bear", "path": "nested/baby_bear" },
+  { "name": "Babiest Bear", "path": "babiest_bear/dataset.json" },
+  { "name": "Goldilocks", "path": "http://example2.com/files/goldilocks" },
+  { "name": "Papa Bear", "path": "http://example3.com/files/papa_bear.json"}
+]
+```
+
+Here's a list of where Nucmorph-Colorizer will check for the manifest files for all of the datasets:
+
+| Dataset      | Expected URL Path                                        |
+| ------------ | -------------------------------------------------------- |
+| Mama Bear    | `http://example.com/data/mama_bear/manifest.json`        |
+| Baby Bear    | `http://example.com/data/nested/baby_bear/manifest.json` |
+| Babiest Bear | `http://example.com/data/babiest_bear/dataset.json`      |
+| Goldilocks   | `http://example2.com/files/goldilocks/manifest.json`     |
+| Papa Bear    | `http://example3.com/files/papa_bear.json`               |
+
+---
+
+</details>
+
+## Getting Started
+
+For most datasets, the easiest way to get started is to modify one of the existing data generation scripts, like [`generate_data.py`](../scripts/timelapse-colorizer-data/generate_data.py).
+
+(Check with your team or one of the developers on the Animated Cell team to see if there's already a data generation script for your project!)
```

Comment:
This might be better placed at the start of the file so that people know immediately whether they need to read the rest or not